### ISSUE TITLE
add new command to delete stale omap objects

### ISSUE THREAD

**Issue by yati1998** [rook/kubectl-rook-ceph#251]
Add a new omap command to list all the stale omap objects and delete them.
Check this for more details:
https://github.com/rook/kubectl-rook-ceph/pull/237#discussion_r1492651612

**Comment by travisn** [comment]
Can you describe the scenario more? Will users commonly hit this issue, or not so common if they are following recommended practices?

**Comment by yati1998** [comment]
There can be cases where the subvolume was deleted but the omaps were not clean, which will result in having a stale omap. Currently, in the deletion of omap we are deleting the omap object and key only for the stale subvolumes to avoid these kinds of scenarios.

**Comment by travisn** [comment]
Sounds good, so we don't expect it to be common, but it may be possible.

**Comment by subhamkrai** [comment]
@yati1998 this is done right?

**Comment by yati1998** [comment]
No, it is not done, it will be different from the subvolume cleanup.
